ManualDeploymentExtended.md

```diff
@@ -0,0 +1,966 @@
+# Extended Manual Setup
+
+This document lists steps for manually deploying BTCPay Server and additional related components. Following these steps is likely to take a long time. A shorter and more pragmatic approach is to use a [docker based deployment](https://github.com/btcpayserver/btcpayserver-docker).
+
+#### Warning: Not recommended for production use
+
+Manual installation is NOT recommended for production use unless you are very confident with your Operating System and Bitcoin security expertise. If you are unsure use [docker](https://github.com/btcpayserver/btcpayserver-docker).
```

Comment:
Same link is directly above. Probably would be more helpful to link one of them to the general Deployment.md in case someone is lost.